crates/sui-proxy/src/main.rs

```diff
@@ -67,15 +67,10 @@ const GIT_REVISION: &str = {
     if let Some(revision) = option_env!("GIT_REVISION") {
         revision
     } else {
-        let version = git_version::git_version!(
+        git_version::git_version!(
```

Comment:
isn't this usually setup with the build.rs ?

Reply:
it's in a few places, but the proxy doesn't build against the sui main binary.  IIRC, three distinct locations use a macro like this.